python/paddle/distributed/checkpoint/load_state_dict.py

```diff
@@ -549,9 +566,11 @@ def load_state_dict(
                         storage_chunk_tensor, src=src_rank, group=process_group
                     )
                 else:
+                    tmp_tensor = paddle.assign(cur_chunk_tensor)
```

Comment:
why use tmp_tensor, plz add comments.

Reply:
The memory held by cur_chunk_tensor may be non-contiguous, and the broadcast API does not support this type of tensor.